neqo-crypto/src/p11.rs

```diff
@@ -47,7 +47,11 @@ macro_rules! scoped_ptr {
             /// # Errors
             ///
             /// When passed a null pointer generates an error.
-            #[allow(clippy::allow_attributes, dead_code, reason = "False positive.")]
+            #[allow(
```

Comment:
```suggestion
            #[expect(
```

Reply:
If I change it to `expect`, I get `warning: this lint expectation is unfulfilled` inside the macro :-(

Reply:
![image](https://github.com/user-attachments/assets/3dcba210-4f76-4192-abf7-16f7e7911313)

There was another one, but that might be the same.

Reply:
Yeah, there are a bunch. Basically, everywhere I used `clippy::allow_attributes` it's because we can't simply do `expect` for some reason or other.